app/models/user.rb

```diff
@@ -175,12 +175,12 @@ def check_permissions_after_save
     end
   end
 
-  def can?(permission, subject = nil)
+  def can?(permission, subject = nil, cache = true)
     if admin?
       true
     else
       @authorizer ||= Authorizer.new(self)
-      @authorizer.can?(permission, subject)
+      @authorizer.can?(permission, subject, cache)
     end
```

Comment:
Maybe you could actually drop a comment here and in the authorizer about the cache argument - why it is there and how to set it (when calling on data that can possibly load too many items in the cache).